magefiles/magefile.go

```diff
@@ -551,19 +550,18 @@ func Shell(ctx context.Context) error {
 		Args("-v", filepath.Join(cwd, ".devcontainer/cache/bash_history")+":/root/.bash_history").
 		Fields("--entrypoint bash").
 		Arg("josephcopenhaver/melody-bot--shell:" + gitsha).
-		New()...).
 		AppendEnv(composeFile).Exec()
 }
 
 func Logs(ctx context.Context) error {
 	mg.Deps(vars)
 
-	argsb := CmdB().Fields("logs -f")
+	args := strings.Fields("logs -f")
 	if s := os.Getenv("SERVICES"); s != "" {
-		argsb.Fields(s)
+		args = append(strings.Fields(strings.TrimSpace(s)))
```

Comment:
oi, args is missing on append func and fields should be varg'd

Reply:
fixed in https://github.com/josephcopenhaver/melody-bot/pull/25